### ISSUE TITLE
A question about training loss

### ISSUE THREAD

**Issue by CaesarYD** [waterdisappear/SAR-JEPA#2]
Excuse me, can I ask how much your loss value reduced to when you fine tune models using 40 shots. My loss value only reduces to about 1.2 with 40 epochs. Some models such as Swim without pre-training are even higher.

**Comment by waterdisappear** [comment]
It is a very strange problem that I have also encountered on some Transformer models. My guess is that some special layer or parameter count makes it difficult to train with small samples, but I couldn't address it adequately due to time constraints. Therefore, I can only give you a small discovery by using LayerNorm before the FC layer, as in linear probing, freezing the other parameters, and then slowly releasing them, as in our latest paper, SARatrX.
